contacts/index.html

```diff
@@ -38,131 +38,41 @@ <h2 class="align-center type-title">Contacts</h2>
-              <button>+</button>
-              <div class="favs-count">0</div>
-              <button>-</button>
-              <img src="images/profile-pics/pic6.jfif" alt="pic1" class="avatar">
-              <input type="checkbox">
-              <span class="full-name">
-                <span>Viorel</span>
-                <span>Popescu</span>
-              </span>
-              <img src="images/trash-1.png" alt="trash" class="trash">
-            </li>
-
-            <li class="person-field">
-              <button>+</button>
-              <div class="favs-count">0</div>
-              <button>-</button>
-              <img src="images/profile-pics/pic7.jfif" alt="pic1" class="avatar">
-              <input type="checkbox">
-              <span class="full-name">
-                <span>Elon</span>
-                <span>Musk</span>
-              </span>
-              <img src="images/trash-1.png" alt="trash" class="trash">
-            </li>
+            -->
           </ul>
-
           <div class="align-center">
-            <button class="button">Delete All</button>
+            <button id="delete-all-button" onclick="deleteAllButtonAction()" class="button">Delete All</button>
           </div>
         </div>
-
         <div class="favourites">
           <h2 class="align-center type-title">Favourites</h2>
-
-          <ol class="list">
-            <li>
-                <p class="align-center">work in progress</p>
-            </li>
-            <li>
-                <p class="align-center">work in progress</p>
-            </li>
-            <li>
-                <p class="align-center">work in progress</p>
-            </li>
-          </ol>
+          <ol id="favs-list" class="list"></ol>
           <div class="align-center">
-            <button class="button">Clear</button>
+            <button id="clear-button" onclick="clearButtonAction()" class="button">Clear</button>
           </div>
         </div>
       </div>
-
-      <form class="add">
+      <form id="add-form" class="add">
         <h2 class="align-center">Add new contact</h2>
         <div>
-          <input name="first_name" placeholder="Your first name" maxlength="20">
+          <input id="first-name" oninput="changeAddButtonState()" name="first_name" placeholder="Your first name" maxlength="20">
         </div>
         <div>
-          <input name="last_name" placeholder="Your last name" maxlength="20">
+          <input id="last-name" oninput="changeAddButtonState()" name="last_name" placeholder="Your last name" maxlength="20">
         </div>
         <div>
-          <input type="url" name="profile_img" placeholder="Your profile image">
+          <input id="profile-image" oninput="changeAddButtonState()" type="url" name="profile_img" placeholder="Your profile image">
         </div>
-        <input type="submit" class="button align-center">
+        <button id="add-button" onclick="addNewPerson()" class="button" type="button" >Add</button>
```

Comment:
form's submit button is type="submit"

and if you can, go with this approach better: https://www.w3schools.com/jsref/event_onsubmit.asp

add onSubmit on the form and the action that should be performed there